add upcoming releses for cert-manager scraper

seemywingz opened the pull request:
<!-- Describe your changes here, include the motivation/context, test coverage, -->
<!-- the type of change i.e. breaking change, new feature, or bug fix -->
<!-- and related GitHub issue or screenshots (if applicable). -->

<!-- Adding a meaningful title and description allows us to better communicate -->
<!-- your work with our users. -->

## Test Plan
<!--- Please describe the tests you have added and your testing environment (if applicable). -->
Ran the scraper locally and verified output 

## Checklist
<!--- Go over all the following points to make sure you've checked all that apply before merging. -->
<!--- If you're unsure about any of these, don't hesitate to ask in our Discord. -->

- [x] If required, I have updated the Plural documentation accordingly.
- [x] I have added tests to cover my changes.
- [x] I have added a meaningful title and summary to convey the impact of this PR to a user.

michaeljguarino commented on static/compatibilities/cert-manager.yaml:
```diff
@@ -3,80 +3,17 @@ git_url: https://github.com/cert-manager/cert-manager
   requirements: []
   incompatibilities: []
   chart_version: 1.12.0
-- version: 1.11.0
```
we shouldn't be removing all these previous versions either

seemywingz replied:
I removed them while testing locally, didn't mean to commit that

michaeljguarino commented:
What code change did you do to change that output?

seemywingz commented:
> What code change did you do to change that output?

I added the "Upcoming Releases" table to be scraped. 


the missing versions was from me testing, I manually removed and added them back to fix the PR
